modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -0,0 +1,222 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-controller-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-autoscaling-ingress-controller_{context}"]
+= Autoscaling an Ingress Controller
+
+Automatically scale an Ingress Controller to dynamically meet routing performance or availablility requirements such as the requirement to increase throughput. The following procedure provides an example for scaling up the default `IngressController`.
+
+.Prequisites
+. You installed the OpenShift CLI (`oc`).
```

Comment:
This is totally up to you, but it might read better as 
```suggestion
. You have the OpenShift CLI (`oc`) installed.
```
Same for the last bullet as well.